packages/lesswrong/components/tagging/TagSubforumPage2.tsx

```diff
@@ -543,27 +546,40 @@ const TagSubforumPage2 = ({classes}: {
     };
     const maxAgeHours = 18;
     const commentsLimit = (currentUser && currentUser.isAdmin) ? 4 : 3;
+
+    const discussionButton = isSubscribed || currentUser?.isAdmin ? (
+      <SectionButton onClick={clickNewDiscussion}>
+        <AddBoxIcon /> <span className={classes.hideOnMobile}>New</span>&nbsp;Discussion
+      </SectionButton>
+    ) : (
+      <LWTooltip title="You must be a member of this subforum to start a discussion" className={classes.newPostLink}>
```

Comment:
This is still not gated behind membership on the backend, but I don't think it's that important. It's more to nudge people to join than to prevent spam etc